From 47df10cd21b93deefe89c1edada47f016ce34a28 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Thu, 24 Nov 2016 11:31:58 -0800
Subject: [PATCH] joyent/node-triton#147 4.15.0 Reviewed by: Todd Whiteman
 <todd.whiteman@joyent.com> Approved by: Todd Whiteman
 <todd.whiteman@joyent.com>

---
 CHANGES.md    | 13 ++++++++++---
 lib/common.js |  2 +-
 package.json  |  2 +-
 3 files changed, 12 insertions(+), 5 deletions(-)

diff --git a/CHANGES.md b/CHANGES.md
index eafca8c..5ce09c9 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -7,9 +7,16 @@ Known issues:
 
 ## not yet released
 
-- [trentm/node-dashdash#30] Change the output used by Bash completion support to
-  indicate "there are no completions for this argument" to cope with different
-  sorting rules on different Bash/platforms. For example:
+(nothing yet)
+
+## 4.15.0
+
+- [joyent/node-triton#64] Support 'triton instance rename ...' (by
+  github.com/YangYong3).
+- [trentm/node-dashdash#30, joyent/node-triton#144] Change the output used by
+  Bash completion support to indicate "there are no completions for this
+  argument" to cope with different sorting rules on different Bash/platforms.
+  For example:
 
         $ triton -p test2 package get <TAB>          # before
         ##-no -tritonpackage- completions-##
diff --git a/lib/common.js b/lib/common.js
index a94ae7e..8f0d84a 100644
--- a/lib/common.js
+++ b/lib/common.js
@@ -90,7 +90,7 @@ function zeroPad(n, width) {
  * raise TypeError trying.
  *
  * @param value {Boolean|String} The input value to convert.
- * @param default_ {Boolean} The default value is `value` is undefined.
+ * @param default_ {Boolean} The default value if `value` is undefined.
  * @param errName {String} The context to quote in the possibly
  *      raised TypeError.
  */
diff --git a/package.json b/package.json
index 07d0307..079b53a 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "triton",
   "description": "Joyent Triton CLI and client (https://www.joyent.com/triton)",
-  "version": "4.14.2",
+  "version": "4.15.0",
   "author": "Joyent (joyent.com)",
   "dependencies": {
     "assert-plus": "0.2.0",
-- 
2.21.0

